packages/cli/src/commands/labs/enable.ts

```diff
@@ -0,0 +1,46 @@
+import color from '@heroku-cli/color'
+import {APIClient, flags, Command} from '@heroku-cli/command'
+import * as Heroku from '@heroku-cli/schema'
+import {Args, ux} from '@oclif/core'
+
+async function enableFeature(heroku: APIClient, feature: string, app?: string) {
+  return heroku.patch<Heroku.AppFeature | Heroku.AccountFeature>(app ? `/apps/${app}/features/${feature}` : `/account/features/${feature}`, {
+    body: {enabled: true},
+  })
+}
+
+export default class LabsEnable extends Command {
+  static description = 'enables an experimental feature'
+  static topic = 'labs'
+
+  static flags = {
+    app: flags.app({required: false}),
+  }
+
+  static args = {
+    feature: Args.string({required: true}),
+  }
+
+  async run() {
+    const {flags, args} = await this.parse(LabsEnable)
+    const feature = args.feature
+    let target
+
+    try {
+      await this.heroku.get<Heroku.AccountFeature>(`/account/features/${feature}`)
+      await enableFeature(this.heroku, feature)
+      const targetResponse = await this.heroku.get<Heroku.Account>('/account')
+      target = targetResponse.body.email
+    } catch (error: any) {
+      if (error.http.statusCode !== 404) throw error
+      // might be an app feature
+      if (!flags.app) throw error
+      await this.heroku.get<Heroku.AppFeature>(`/apps/${flags.app}/features/${feature}`)
+      await enableFeature(this.heroku, feature, flags.app)
+      target = flags.app
+    }
+
+    ux.action.start(`Enabling ${color.green(feature)} for ${color.cyan(target!)}`)
```

Comment:
This should be moved to before the requests are made. Right before the `try` looks right to me. The original started it at the `enableFeature` call, but should have done it sooner.

Reply:
@ryandagg  Just realized that target can't be set without making the calls to the specified endpoints. Unless we default the `ux.action.start` string to be `flags.app`, but I don't think that's the default behavior. Thoughts?

Reply:
This whole setup is weird. See suggestion.